.github/workflows/master.yml

```
@@ -241,6 +241,11 @@ jobs:
     if: github.event_name != 'pull_request' || github.event.pull_request.draft == false
     runs-on: ubuntu-latest
     timeout-minutes: 120
+    strategy:
+      matrix:
+        group: [1, 2, 3]
+        include:
+          - splits: 3
```

```suggestion
    strategy:
      matrix:
        group: [1, 2]
        include:
          - splits: 2
```

Can we decrease `splits`?

---

@harupy 
Yes.
GTW, is there a target execution time?
It's taking around 30 minutes even when split into three parts.

---

Never mind. 3 is fine.

---

Sure, maybe increasing the number might be fine if we want to shorten the execution time.